### DISCUSSION TITLE
[@types/jest] Export a `mocked` function

### DISCUSSION THREAD

**Discussion by samhwang** [DefinitelyTyped/DefinitelyTyped#57716] [Issues with a @types package]
The `mocked` function from `ts-jest` is now merged into the Jest tool chain (from Jest v24 onwards), and will be deprecated in `ts-jest` v28. This function will now be exported from the `jest` package.

Related Issue in Jest Repo here:
https://github.com/facebook/jest/issues/12131

[a comment by an automated account is not shown]

**Comment by G-Rath** [comment]
Have opened #57775 for this
